loopchain/consensus/consensus.py

```diff
@@ -16,29 +16,44 @@
 import logging
 import threading
 import time
+from collections import namedtuple
 
 from loopchain import configure as conf
 from loopchain import utils as util
 
 from loopchain.blockchain import Block, TransactionStatusInQueue
 from loopchain.channel.channel_property import ChannelProperty
 from loopchain.consensus import Publisher, Epoch, EpochStatus
-from loopchain.baseservice import ObjectManager, CommonThread, Timer
+from loopchain.baseservice import ObjectManager, CommonThread, Timer, BlockGenerationScheduler
 from loopchain.baseservice.aging_cache import AgingCache
 
 
 class Consensus(CommonThread, Publisher):
+    EVENT_COMPLETE_CONSENSUS = "complete_consensus"
+    EVENT_MAKE_BLOCK = "make_block"
+    EVENT_LEADER_COMPLAIN_F_1 = "leader_complain_f_1"
+    EVENT_LEADER_COMPLAIN_F_2 = "leader_complain_2f_1"
```

Comment:
Why are event name(`F_2`) different from value(`2f_1`)?

Reply:
I gave the name to the variable like that just for convenience. I got your comment that the naming occurs confuse to others who don't know well about this feature. I'll apply to code your suggestion. Thank you :)